packages/govern/internal/clients/graphql/GraphQLClient.ts

```diff
@@ -0,0 +1,103 @@
+import fetch from 'isomorphic-unfetch'
+import { Client } from '@urql/core'
+import { DocumentNode } from 'graphql'
+import ClientInterface from '../lib/ClientInterface'
+
+/**
+ * TODO: Use QueryResult type from govern-server/core
+ *
+ * @class GraphQLClient
+ */
+export default class GraphQLClient implements ClientInterface {
+  /**
+   * The urql/core Client object
+   *
+   * @var {Client} client
+   *
+   * @private
+   */
+  private client: Client
```

Comment:
Should we rename `Client` into `UrqlClient` for clarity?

```suggestion
  private urlqClient: UrqlClient
```

Reply:
I think the comment brings up enough clarity :)